packages/editor-ui/src/plugins/codemirror/completions/utils.ts

```diff
@@ -43,9 +43,8 @@ export function longestCommonPrefix(...strings: string[]) {
 // suggestions can be matched based on it.
 function extractSubExpression(userInput: string): string {
 	const dollarSignIndex = userInput.indexOf('$');
-	// If it's not a dollar sign expression just strip parentheses
 	if (dollarSignIndex === -1) {
-		userInput = userInput.replace(/^.+(\(|\[|{)/, '');
```

Comment:
Went through the referenced PR, but I cannot quite tell why we needed to strip parens here. Since parens are needed for indexed access expressions and since all tests still pass, it looks safe to remove. Else let me know.

/cc @MiloradFilipovic

Reply:
38018b9735